### DISCUSSION TITLE
Difference Between GE-Proton & GE-1

### DISCUSSION THREAD

**Discussion by trymeouteh** [DavidoTek/ProtonUp-Qt#91] [Q&A]
What is the difference between GE-Proton7-3 and 7.3-GE-1? Are there two different kinds of types of Proton-GE?

**Comment by begin-theadventure** [comment]
[GE-Proton7-3](https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/GE-Proton7-3) and [7.3-GE-1](https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/7.3-GE-1).

**Reply by trymeouteh** [reply]
Is there 7.3-1, 7.3-2 and 7.3-3 with 7.3-3 being the final version of 7.3?

**Reply by DavidoTek** [reply]
They have changed the naming scheme from `Proton-X.Y-GE-Z` to `GE-ProtonA-B`.
Have a look at https://github.com/GloriousEggroll/proton-ge-custom/releases, there is a list of all versions, showing which one is the newest (release date)
